editor.planx.uk/src/pages/FlowEditor/lib/analytics/types.ts

```diff
@@ -108,6 +111,11 @@ type HelpClick = {
   metadata: HelpClickMetadata;
 };
 
+type SaveClick = {
```

Comment:
The other events have a line or two in jsdoc explaining their use. I appreciate that this seems simple and straightforward, but there's a decent level of complexity and context here so it's worth maintaining this pattern.